File: Spider/Spider/SpiderCleanUp.vb
Imports Contensive.BaseClasses
Public Class SpiderCleanUp
    Inherits AddonBaseClass
    Public Overrides Function Execute(ByVal CP As CPBaseClass) As Object


        CP.Db.ExecuteNonQuery("delete from ccspiderdocs from ccspiderdocs  left join cclinkaliases on ccSpiderDocs.pageid = cclinkaliases.pageid  where (ccSpiderDocs.pageid <> 0) and ccLinkAliases.id is null")

    End Function

End Class

File: server/Spider/SpiderCleanUp.cs
using Contensive.BaseClasses;

namespace Contensive.Addons.Spider {
    public class SpiderCleanUp : AddonBaseClass {
        public override object Execute(CPBaseClass CP) {


            CP.Db.ExecuteNonQuery("delete from ccspiderdocs from ccspiderdocs  left join cclinkaliases on ccSpiderDocs.pageid = cclinkaliases.pageid  where (ccSpiderDocs.pageid <> 0) and ccLinkAliases.id is null");
            return default;

        }

    }
}